src/components/Datagrid/customCells/Money/utils.test.ts

```diff
@@ -0,0 +1,12 @@
+import { hasDiscountValue } from "./utils";
+
+describe("MoneyCell utils", () => {
+  describe("hasDiscountValue", () => {
+    it("should return true if value is not undefined, null or empty string", () => {
```

Comment:
`hasDiscountvalue` doesn't take strings (should it?)
```suggestion
    it("should return true if value is not undefined, null", () => {
```